backend/app/main.py

```diff
@@ -32,4 +41,41 @@ def custom_generate_unique_id(route: APIRoute) -> str:
         allow_headers=["*"],
     )
 
+
+def get_current_user():
```

Comment:
This code should definitely not be in main.py. Remember, each service (e.g. inquiry service) should have its own dedicated file, with just the code for that service. That'll make it much easier to read the code and understand where everything is.

Reply:
refactored